project/Dependencies.scala

```diff
@@ -41,7 +41,7 @@ object Dependencies {
     "com.typesafe.scala-logging"    %% "scala-logging"       % "3.9.5",
 
     "org.parboiled" % "parboiled-core" % "1.4.1",
-    excludeGuava("org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-e0584dbdc")
+    excludeGuava("org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-34e8799b-SNAP")
```

Comment:
So does the Rawls PR have to merge first, and then the version number gets updated here?

Reply:
As far as I know, I have to merge this PR first then merge the Rawls PR and update with an official version here but I could be wrong